src/main/java/com/iota/iri/conf/SnapshotConfig.java

```diff
@@ -6,72 +6,99 @@
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_PRUNING_DELAY}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_PRUNING_DELAY}
      */
     int getLocalSnapshotsPruningDelay();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_INTERVAL_SYNCED}
      */
     int getLocalSnapshotsIntervalSynced();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED}
      */
     int getLocalSnapshotsIntervalUnsynced();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_DEPTH}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_DEPTH}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_DEPTH}
      */
     int getLocalSnapshotsDepth();
 
     /**
-     * @return {@value Descriptions#SNAPSHOT_TIME}
+     * Default Value: {@value BaseIotaConfig.Defaults#SNAPSHOT_TIME}
+     *
+     * @return {@value SnapshotConfig.Descriptions#SNAPSHOT_TIME}
      */
     long getSnapshotTime();
 
     /**
-     * @return {@value Descriptions#SNAPSHOT_FILE}
+     * Default Value: {@value BaseIotaConfig.Defaults#SNAPSHOT_FILE}
+     *
+     * return {@value SnapshotConfig.Descriptions#SNAPSHOT_FILE}
      */
     String getSnapshotFile();
 
     /**
-     * @return {@value Descriptions#SNAPSHOT_SIGNATURE_FILE}
+     * Default Value: {@value BaseIotaConfig.Defaults#SNAPSHOT_SIGNATURE_FILE}
+     *
+     * @return {@value SnapshotConfig.Descriptions#SNAPSHOT_SIGNATURE_FILE}
      */
     String getSnapshotSignatureFile();
 
     /**
-     * @return {@value Descriptions#MILESTONE_START_INDEX}
+     * Default Value: {@value BaseIotaConfig.Defaults#MILESTONE_START_INDEX}
+     *
+     * @return {@value SnapshotConfig.Descriptions#MILESTONE_START_INDEX}
      */
     int getMilestoneStartIndex();
 
     /**
-     * @return {@value Descriptions#LOCAL_SNAPSHOTS_BASE_PATH}
+     * Default Value: {@value BaseIotaConfig.Defaults#LOCAL_SNAPSHOTS_BASE_PATH}
+     *
+     * @return {@value SnapshotConfig.Descriptions#LOCAL_SNAPSHOTS_BASE_PATH}
      */
     String getLocalSnapshotsBasePath();
 
-
     /**
-     * @return {@value Descriptions#PREVIOUS_EPOCH_SPENT_ADDRESSES_FILE}
+     * Default Value: {@value BaseIotaConfig.Defaults#PREVIOUS_EPOCHS_SPENT_ADDRESSES_FILE}
+     *
+     * @return {@value SnapshotConfig.Descriptions#PREVIOUS_EPOCH_SPENT_ADDRESSES_FILE}
      */
     String getPreviousEpochSpentAddressesFiles();
 
     /**
+     * Default Value: {@value BaseIotaConfig.Defaults#SPENT_ADDRESSES_DB_PATH}
+     *
      * @return {@value Descriptions#SPENT_ADDRESSES_DB_PATH}
```

Comment:
Normalize to `SnapshotConfig.Descriptions.`

Reply:
Why? Since `Descriptions` is inside `SnapshotConfig` it is clear enough which one we use I think.

I believe that in the Style Guide I wrote something against using static imports and using full names instead. This is not in the scope of what was described in the style guide.

Reply:
Why did you use `SnapshotConfig.Descriptions` for the other getters?

Reply:
Now you made me look and I remember why...

When I try to read the javadocs with IntelliJ there are issues if I don't use the fully qualified name
So I will change, thanks for the catch